src/main/java/com/ezasm/gui/Window.java

```diff
@@ -15,6 +16,13 @@
  * relevant simulator.
  */
 public class Window {
+    public class School {
```

Comment:
What is this and why does it exist? My LSP finds no references to this anywhere in the project.

Reply:
got rid of it, ik how it got there and its quite embarrassing and id prefer not to say how nor why.